tests/test-cache-ttl-manager-rest.php

```diff
@@ -78,22 +76,9 @@ public function test__set_ttl_for_unauthenticated_write_requests( $method ) {
 
 	/**
 	 * @dataProvider get_rest_read_methods
-	 */
-	public function test__skip_ttl_for_authenticated_read_requests( $method ) {
-		$user_id = $this->factory->user->create();
-		wp_set_current_user( $user_id );
-
-		$response = $this->dispatch_request( $method );
-
-		$response_headers = $response->get_headers();
-
-		$this->assertArrayNotHasKey( 'Cache-Control', $response_headers );
-	}
-
-	/**
 	 * @dataProvider get_rest_write_methods
 	 */
-	public function test__skip_ttl_for_authenticated_write_requests( $method ) {
+	public function test__skip_ttl_for_authenticated_requests( $method ) {
```

Comment:
`test__skip_ttl_for_authenticated_read_requests()` and `test__skip_ttl_for_authenticated_write_requests()` had identical implementation